kevlar/simlike.py

```diff
@@ -282,9 +286,9 @@ def main(args):
         args.sample_labels = default_sample_labels(nsamples)
 
     kevlar.plog('[kevlar::simlike] Loading k-mer counts for each sample')
-    case = khmer.Counttable.load(args.case)
-    controls = [khmer.Counttable.load(c) for c in args.controls]
-    refr = khmer.SmallCounttable.load(args.refr)
+    case = kevlar.sketch.load(args.case)
+    controls = [kevlar.sketch.load(c) for c in args.controls]
+    refr = kevlar.sketch.load(args.refr)
```

Comment:
Auto-detect sketch type and load appropriately, rather than prescribing sketch type in the code.